components/message.rst

```diff
@@ -0,0 +1,284 @@
+.. index::
+   single: Message
+   single: Components; Message
+
+The Message Component
+=====================
+
+    The Message component helps application to send and receive messages
+    to/from other applications or via
```

Comment:
This phrase looks unfinished. Also, it's not a very descriptive description. Should we mention at least "message bus" or some key concept so the readers know undoubtely what problem does this component solve? Thanks!